src/wallet/wallet.cpp

```diff
@@ -1511,7 +1516,7 @@ bool CWalletTx::RelayWalletTransaction()
     {
         if (GetDepthInMainChain() == 0 && !isAbandoned() && InMempool())
         {
-            LOGA("Relaying wtx %s\n", GetHash().ToString());
+            // LOGA("Relaying wtx %s\n", GetHash().ToString());
```

Comment:
what about using a restrictive debug category rather then LOGA and uncomment the log line?

Reply:
I think we need a general WALLET debug category and the SELECTCOINS logs should be moved to it.  But I think that this should be done in another PR.

Reply:
agreed